src/pages/Explorer/Category/UserCreation.svelte

```diff
@@ -50,7 +50,7 @@
   </div>
 </a>
 
-<div class="bottom row justify v-center c-waterloo mrg-s mrg--t" class:caption={small}>
+<div class="bottom row justify v-center c-waterloo mrg-s mrg--t" class:caption={small} href={url} target="_blank" on:click={onClick}>
```

Comment:
`div` elements don't have the `href` and `target` attributes. Handle the link behaviour inside the `onClick` handler

Reply:
replaced `div` with `a` tag